slf4j-api/pom.xml

```diff
@@ -60,11 +60,13 @@
         <configuration>
           <instructions>
             <Import-Package>org.slf4j.spi;version="${range;[===,+);${version_cleanup;${project.version}}}"</Import-Package>
-            <!-- Export the client/user package of slf4j-api version 1 to make the slf4j-api bundle in version 2 usable for bundles that only import slf4j-1.x -->
-            <_exportcontents><![CDATA[
+            <!-- Export the client/user packages of slf4j-api in version 1 to make the slf4j-api bundle in version 2 usable for bundles that only import version ranges not including 2.x -->
+            <!-- Uses directive not automatically added: https://github.com/bndtools/bnd/issues/5789 but in this case not necessary either -->
+            <Export-Package><![CDATA[
               *,\
-              org.slf4j;uses:="org.slf4j.event,org.slf4j.helpers,org.slf4j.spi";version="${latest.1.version}"
-            ]]></_exportcontents>
+              org.slf4j;version="${backwards.compatible.version}",\
```

Comment:
Since https://github.com/bndtools/bnd/issues/5789 was closed should we add the uses constraints back?
But I haven't read Peter Kriens answer in full detail.

Reply:
Basically it says, that the tool does not support emitting uses twice (automatically generated) but in this particular case they are not necessary either (as there won't be ever any other bundle exporting those packages from the uses directive). Therefore I removed the manually copied uses constraints.